plugins/filter_parser/filter_parser.c

```diff
@@ -292,6 +292,7 @@ static int cb_parser_filter(const void *data, size_t bytes,
                                     append_arr_len--;
                                     append_arr[append_arr_i] = NULL;
                                 }
+                                break;
```

Comment:
It seems like you can place `break;` to 301 line and remove this break and one in else block.

Reply:
Thank you for reviewing. I updated the patch.